### DISCUSSION TITLE
Announcement about restore-dev branch progress

### DISCUSSION THREAD

**Discussion by Tylerjet** [Staubgeborener/klipper-backup#113] [General]
Hey All,

Just wanted to make a small little post here, I already let Staubgeborener know that progess on the restore branch will be on hold for a bit. As of last week I am currently searching for a new Job as my current Job will cease on the 31st of August. 

Most of my focus of course will now be on Job hunting. I may still post replies here and there to issue tickets but will likely not be making any progress or work on the restore branch until new employment is found.

~MrTylerjet

**Comment by Tylerjet** [comment]
Update*

New job has been found (thank goodness) hopefully in the coming months (likely some time after christmas or sooner if I have a chance to work a bit on it during my thankgiving vacation) I can finish up work on the restore branch so it can be moved to beta. Just need a bit more time to get settled in and comfortable with the new job first. 😄

**Reply by tismofied** [reply]
Congratulations on the new job!!!

**Comment by Tylerjet** [comment]
![GIF](https://media4.giphy.com/media/v1.Y2lkPTc5MGI3NjExeTR0bHpueDFmOHBrcGl6M3hsNzdkMXhvd2RnY2QwMWw5cnF4eHo2aiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/ncU3bkZ5ghDlS/giphy.gif)
